### ISSUE TITLE
MacOS Ventura 13 does not work WSO2 Integration Studio 8 and 8.1.

### ISSUE THREAD

**Issue by fosbsb** [wso2/api-manager#1006]
### Description

After updating, Integration Studio no longer works, it is not even possible to update, because as soon as you open it, it closes.

![image](https://user-images.githubusercontent.com/3026559/203564932-1027c34b-ddcf-4f79-b527-cd1d1e0fbc09.png)

### Steps to Reproduce

When opening Integration Studio, chosen Workspace, the initial screen is displayed and closes.

It's not even possible to update the packages to see if it resolves.

### Affected Component

IntegrationStudio

### Version

8.1.0

### Environment Details (with versions)

MacOS Ventura 13.0.1 (22A400)

java version "11.0.9" 2020-10-20 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.9+7-LTS)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.9+7-LTS, mixed mode)

### Relevant Log Output

```shell
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               IntegrationStudio [7330]
Path:                  /Applications/IntegrationStudio 8.1.0.app/Contents/MacOS/IntegrationStudio
Identifier:            WSO2-Integration-Studio
Version:               8.1.0 (8.1.0.202203281342)
Code Type:             X86-64 (Native)
Parent Process:        launchd [1]
User ID:               502

Date/Time:             2022-11-21 03:13:23.5866 -0300
OS Version:            macOS 13.0.1 (22A400)
Report Version:        12
Bridge OS Version:     7.0 (20P420)
Anonymous UUID:        62E89E18-244E-340C-9848-0286032B8D9E


Time Awake Since Boot: 11000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000

Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
Terminating Process:   exc handler [7330]
```

### Suggested Labels

wso2 enterprise-integratorwso2 integration-studio

**Comment by tsega** [comment]
Any solution for this?

**Comment by fosbsb** [comment]
So far I haven't had any feedback.

**Comment by lukinsj** [comment]
I have the same issue as well.  Looking forward to a timely solution.

**Comment by lakshithagunasekara** [comment]
Can anyone try whether answer in this[1] works?  

[1] - https://stackoverflow.com/questions/74514999/wso2-integration-studio-not-work-macos-ventura-13

**Comment by rosensilva** [comment]
Apple has released macOS Ventura 13.2 and it is available for intel MacBooks. With this macOS update the integration studio is working as expected. 
Related Apple issue: https://bugs.webkit.org/show_bug.cgi?id=247387
